docs/source/reference/rest.md

```diff
@@ -136,8 +136,8 @@ Authorization header.
 ### Use requests
 
 Using the popular Python [requests](https://docs.python-requests.org)
-library, here's example code to make an API request for the users of a JupyterHub
-deployment. An API GET request is made, and the request sends an API token for
+library, (here's example code to make an API request for the users of a JupyterHub
+deployment) an API GET request is made, and the request sends an API token for
```

Comment:
I don't think a parenthetical helps clarity here. If we have a problem, maybe move the 'here's example...' bit to its own sentence at the end?